src/layout/FeaturesSection/FeaturesSection.svelte

```diff
@@ -31,29 +46,11 @@
 			title: $_("home.features.tags.title"),
 			description: $_("home.features.tags.description"),
 			icon: Tag
-		},
-		{
-			title: $_("home.features.tabs.title"),
-			description: $_("home.features.tabs.description"),
-			icon: TabDesktop
 		}
 	] as const satisfies readonly FeatureCardData[];
-
-	onMount(() => {
-		if (!dev) {
-			// Iterate through feature cards every n seconds
-			setInterval(() => {
-				currentFeature += currentFeature !== 3 ? 1 : -currentFeature;
-				cardPaginationInterval = 16;
-			}, cardPaginationInterval * 1000);
-		}
-	});
 </script>
 
 <PageSection id="features-section">
-	<div class="features-section-left">
-		<FeatureShowcase feature={currentFeature} />
```

Comment:
if you're getting rid of this here, it's best to clean up the other components related to this as well.

Reply:
I have an idea to repurpose them so let's keep them around for now.